pkg/model/master_volumes.go

```diff
@@ -247,9 +247,6 @@ func (b *MasterVolumeBuilder) addGCEVolume(c *fi.ModelBuilderContext, name strin
 
 func (b *MasterVolumeBuilder) addOpenstackVolume(c *fi.ModelBuilderContext, name string, volumeSize int32, zone string, etcd kops.EtcdClusterSpec, m kops.EtcdMemberSpec, allMembers []string) error {
 	volumeType := fi.StringValue(m.VolumeType)
-	if volumeType == "" {
```

Comment:
if you remove this, then we should not either post volumetype to openstack APIs. https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/openstacktasks/volume.go#L134

I know that openstack uses default type if you do not define anything. However, this modification is not enough - we should not post anything to openstack api if we do not have volumetype defined

Reply:
My reasoning for only changing this particular code is:
`cinderv3.CreateOpts.VolumeType` is not a pointer so we don't change anything by conditionally setting that. The serialisation in gophercloud also uses `omitempty` so empty strings will not be serialised to the request payload.